.github/workflows/run-tests.yaml

```diff
@@ -11,10 +11,16 @@ on:
     paths-ignore:
       - 'apple/**'
 
+env:
+  NEXT_PUBLIC_APP_ENV: prod
+  NEXT_PUBLIC_BASE_URL: http://localhost:3000
+  NEXT_PUBLIC_SERVER_BASE_URL: http://localhost:4000
+  NEXT_PUBLIC_HIGHLIGHTS_BASE_URL: http://localhost:3000
+
 jobs:
   run-code-tests:
     name: Run Codebase tests
-    runs-on: ubuntu-latest-m
```

Comment:
Do you know why this change was needed? The `-m` should be telling the worker to use the larger sized instances for this job.

Reply:
Ahh i see this is for forked workflows. I wonder how we can handle this, as the smaller instances can timeout on us in the main repo